tools/mtouch/Application.cs

```diff
@@ -1995,6 +1995,10 @@ public void BuildMSymDirectory ()
 			foreach (var target in Targets) {
 				GenerateMSymManifest (target, target_directory);
 				var msymdir = Path.Combine (target.BuildDirectory, "Msym");
+				if (!Directory.Exists (msymdir)) {
+					ErrorHelper.Warning (95, $"Aot files could not be copied to msym directory is missing: '{msymdir}'");
```

Comment:
The error message is hard to read.
The error code must be unique, use a new one and add it on the documentation files.